bcgov/biohubbc-platform
# SIMSBIOHUB-73: Project Details Page Update

# Overview
This PR requires the changes in this PR: [PR-1044](https://github.com/bcgov/biohubbc/pull/1044)

## Links to Jira tickets
[SIMSBIOHUB-73](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-73)
[UXPin](https://preview.uxpin.com/6d6b732a0a737e36bcbf5874b64311858572ca6c#/pages/162852443/specification/no-panels)

## Description of relevant changes
- Broke down handlebars template to handle more complex UI
- Updated data shown in details section of Project Details Page
- Updated overall layout of Project Details Page
- Added configuration props to the Map Container

## Testing Instructions
1. Create a project in SIMS (running the code from this PR)
- Add activities to this project
- Add IUCN Conservation Actions to this project
- Add funding sources to this project
- Add partners to this project
2. Open BioHub and search for the newly created project
3. Open details page for that project and see new UI/ changes

## PR Checklist

A list of items that are good to consider when making any changes.

_Note: this list is not exhaustive, and not all items are always applicable._

### Code


### Style


### Documentation


### Tests


### Linting/Formatting

       _See the `lint` commands in package.json_
       _See the `format` commands in package.json_

### SonarCloud
